Backoffice/Model/GroupInterface.php

```diff
@@ -48,26 +48,29 @@ public function getLabels();
     /**
      * @return array
      */
-    public function getNodeRoles();
+    public function getModelRoles();
 
     /**
-     * @param NodeGroupRoleInterface $nodeGroupRole
+     * @param ModelGroupRoleInterface $modelGroupRole
      */
-    public function addNodeRole(NodeGroupRoleInterface $nodeGroupRole);
+    public function addModelRole(ModelGroupRoleInterface $modelGroupRole);
 
     /**
-     * @param string $nodeId
+     * @param string $type
+     * @param string $id
      * @param string $role
      *
-     * @return NodeGroupRoleInterface|null
+     * @return ModelGroupRoleInterface|null
      */
-    public function getNodeRoleByNodeAndRole($nodeId, $role);
+    public function getModelRoleByTypeAndIdAndRole($type, $id, $role);
```

Comment:
We get a ModelGroupRole so `getModelGroupRoleByTypeAndIdAndRole($type, $id, $role)` sounds better.